src/test/resources/projects/forge/1206/build.gradle

```diff
@@ -18,6 +18,17 @@ group = project.maven_group
 def mcVersion = "@MCVERSION@"
 def forgeVersion = "@FORGEVERSION@"
 
+
+loom {
+	forge {
+		mixinConfig "examplemod.mixins.json"
+	}
+
+	afterEvaluate {
+		assert mixin.useLegacyMixinAp.get() == false
```

Comment:
Does the test kit run Gradle with assertions enabled (= does this assertion run)?

Reply:
I tested a random `assert false`, seems to be working fine.
<img width="1590" alt="image" src="https://github.com/user-attachments/assets/9c34763c-7ca7-4cde-a36f-65e5554af383">